docs/umberto.json

```diff
@@ -299,6 +299,7 @@
 		"updating/migration-from-ckeditor-4.html#migration-from-ckeditor-4-faq": "updating/ckeditor4/ckeditor4-troubleshooting.html",
 		"updating/migration-from-ckeditor-4.html#plugins-compatibility-table": "updating/ckeditor4/ckeditor4-plugin-compatibility.html",
 		"updating/migration-from-ckeditor-4.html#configuration-options-compatibility-table": "updating/ckeditor4/ckeditor4-configuration-compatibility.html",
+		"updating/migration-from-cke4.html#configuration-options-compatibility-table": "updating/ckeditor4/ckeditor4-configuration-compatibility.html",
```

Comment:
How did this change get in here?

Reply:
Don't know. Probably wrong auto merge. Will be changed